.vscode/tasks.json

```diff
@@ -3,7 +3,9 @@
   "tasks": [
     {
       "type": "shell",
-      "command": "yarn start",
+      "osx": {
```

Comment:
We don't want to auto-run this in the cloud VSCode environment, so changing the setting to only run automatically in our MacOS environment (should be no change on our end)